query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedConnectMutationInsideCreateSpec.scala

```diff
@@ -961,8 +962,8 @@ class NestedConnectMutationInsideCreateSpec extends FlatSpec with Matchers with
          |}
       """.stripMargin,
       project,
-      errorCode = 3039,
-      errorContains = s"No Node for the model Comment with value $todoId for id found."
+      errorCode = 0, // 3039,
+      errorContains = """RecordNotFound(\"Expected 1 records to be connected, found 0.\")"""
```

Comment:
It feels like it should be an instance of a user facing "NotFound" error.

Reply:
yes :+1